DropboxLike.Domain/Repositories/User/UserRepository.cs

```diff
@@ -16,15 +16,15 @@ public class UserRepository : IUserRepository
 {
     private readonly ApplicationDbContext _applicationDbContext;
     private readonly IAmazonS3 _awsS3Client;
-    private readonly string _bucketName;
+    private readonly string? _bucketName;
 
 
     public UserRepository(ApplicationDbContext applicationDbContext, IOptions<AwsConfiguration> options)
     {
         var configuration = options.Value;
         _applicationDbContext = applicationDbContext;
         _bucketName = configuration.BucketName;
-        _awsS3Client = new AmazonS3Client(configuration.AwsAccessKey, configuration.AwsSecretAccessKey, RegionEndpoint.GetBySystemName(configuration.Region));
+        _awsS3Client = new AmazonS3Client(configuration.AwsAccessKey, configuration.AwsSecretAccessKey, RegionEndpoint.USEast1); //.GetBySystemName(configuration.Region));
```

Comment:
This is also a 'bad' change because this was working before as part of loading configuration from environment variables.